components/servo/Cargo.lock

```diff
@@ -1526,6 +1525,7 @@ dependencies = [
  "log 0.3.6 (registry+https://github.com/rust-lang/crates.io-index)",
  "msg 0.0.1",
  "num-traits 0.1.35 (registry+https://github.com/rust-lang/crates.io-index)",
+ "regex 0.1.76 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
We'll need to run `./mach cargo-update -p net_traits` to ensure that ports/cef/Cargo.lock is updated too.